session6/com.itschool.course/src/homework/Ex18.java

```diff
@@ -0,0 +1,13 @@
+package homework;
+
+public class Ex18 {
+
+    public static void main(String[] args) {
+
+        int[] myArray = new int[]{1, 2, 3, 4, 5};
+
+        for(int i = 0; i <=5; i++){
+            System.out.println(myArray[i]);
```

Comment:
this will just print the elements of the array, not their incremented positions. Change the value of the elements in the array to be more clear. However, when running it, I get an error `Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 5`
Can you tell why?